.springBeans

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this file be added in `.gitignore`?

Reply:
Done!